Song/Songwriter.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class SongWriter
+    {
+        private StringBuilder _song;
+        private VerseWriter _verseWriter;
```

Comment:
VerseWrite is stateful. If you sing a song twice - it will be messed up. YOu need to reset the state at the start of sing or after you sung.

Reply:
The verseWriter is stateful on purpose - since adding a writing a new verse just means adding two lines to the previous verse, there's no need for the verseWriter to redo the entire verse again, instead it can just add the two lines needed and append the new verse to the song string. If the user needs to write a entirely new song from scratch they can create a new SongWriter instance (or I can add a Clear() function). I also added a ToString() method for the SongWriter class if the user should need to view the song contents without modifying them.